cli/commands/CreateOrderCommand.php

```diff
@@ -0,0 +1,81 @@
+<?php
+namespace Pizzaservice\cli\commands;
+
+use Order;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Console\Command\Command;
+
+/**
+ * Implementation of a command where the user is going to be asked to create new order-values which are going to be
+ * included into the order-table of the pizzaService-database.
+ *
+ * @class CreateOrderCommand
+ */
+class CreateOrderCommand extends Command
+{
+    protected static $defaultName = "create:order";
+
+    /**
+     * Configure additional instances.
+     *
+     * A description about the function of the command.
+     *
+     * @return void
+     */
+    protected function configure()
+    {
+        $this->setDescription("Sets new order values which are going to be included into the order-table.\n");
+    }
+
+    /**
+     * Creates and includes new order-values into the order-table.
+     *
+     * The user is going to be asked about new order values which are going to be included inside
+     * order-table after the confirmation question was confirmed.
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return int
+     * @throws \PropelException
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
```

Comment:
Should'nt an order also include some references to pizzas that are contained in the order?
That was also the reason why I did not add that as a requirement in Phase 3.
The commandline-interface is thought of the admin interface that only the pizza vendor needs.